draft-packages/card/docs/Card.stories.tsx

```diff
@@ -1,13 +1,13 @@
 import React from "react"
-import { Box, Heading } from "@kaizen/component-library"
+import { Story } from "@storybook/react"
 import { withDesign } from "storybook-addon-designs"
-import { Card, CardProps } from ".."
+import { Card } from ".."
 import { figmaEmbed } from "../../../storybook/helpers"
 import { CATEGORIES } from "../../../storybook/constants"
-import cardStoriesData from "./card-stories-data.json"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
 
 export default {
-  title: `${CATEGORIES.components}/Card`,
+  title: `${CATEGORIES.components}/Container`,
```

Comment:
Curious about the decision to rename this here now. I know it's been renamed in the UI Kit, but I was thinking we'd change this over when we change the component name as well so that it's not one thing in storybook and another thing in code.

Reply:
Yeah thats probably best, less confusion.